pinot-server/src/main/java/org/apache/pinot/server/api/resources/TablesResource.java

```diff
@@ -436,7 +436,8 @@ public Response downloadSegment(
   public String uploadLLCSegment(
       @ApiParam(value = "Name of the REALTIME table", required = true) @PathParam("realtimeTableName")
           String realtimeTableName,
-      @ApiParam(value = "Name of the segment", required = true) @PathParam("segmentName") String segmentName)
+      @ApiParam(value = "Name of the segment", required = true) @PathParam("segmentName") String segmentName,
+      @ApiParam(value = "Whether to update hdfs url") @QueryParam("updateUrl") boolean updateUrl)
```

Comment:
What is the default value here if users do not specify it?

Reply:
thanks for review. let me add defaultValue for this.